config/locales/en.yml

```diff
@@ -30,6 +30,30 @@ en:
       sign_up: Sign Up
       sign_in: Sign In
       sign_out: Sign Out
+  layouts:
+    global_announcements:
+      close: Close
+      more_announcements: More Announcements...
+  admin:
+    navbar:
+      announcements: Announcements
+    admin:
+      home:
+        message: "Pages"
+    announcements:
+      index:
+        header: "Announcements"
```

Comment:
Maybe we should just accept this first, merge it, then replace all to single quotes in one fell swoop. the other files are missing convention also.